src/tests/Microsoft.PowerFx.Interpreter.Tests/PowerFxEvaluationTests.cs

```diff
@@ -18,7 +18,8 @@ public class ExpressionEvaluationTests : PowerFxTest
     {
         internal static Dictionary<string, Func<PowerFxConfig, (RecalcEngine engine, RecordValue parameters)>> SetupHandlers = new ()
         {
-            { "OptionSetTestSetup", OptionSetTestSetup }
+            { "OptionSetTestSetup", OptionSetTestSetup },
+            { "PatchTestSetup", PatchTestSetup }
```

Comment:
>PatchTestSetup [](http://example.com/codeflow?start=15&length=14)

Can we reuse this to for .txt files for Collect ?

Reply:
Yes. I'll change the handler's name to something more generic.

Reply:
Collect test cases have been moved to Collect.txt.